enterprise/backend/src/metabase_enterprise/workspaces/api.clj

```diff
@@ -460,7 +461,12 @@
             [:target {:optional true} ::transform-target]]]
   (api/check-404 (t2/select-one :model/WorkspaceTransform :ref_id tx-id :workspace_id id))
   (t2/update! :model/WorkspaceTransform tx-id body)
-  (fetch-ws-transform id tx-id))
+  (let [transform (fetch-ws-transform id tx-id)]
+    ;; Re-sync dependencies if source or target changed
+    (when (or (:source body) (:target body))
```

Comment:
I suspect the FE will send these even when they haven't changed.

This is fine for now though.